### DISCUSSION TITLE
ctx.stop_loss = ATR

### DISCUSSION THREAD

**Discussion by ghost** [edtechre/pybroker#175] [Q&A]
Hi,

Does anyone know if the stop loss calculated off the open price entry if it's ctx.buy_fill_price = PriceType.OPEN or off the signal bar's close?

Thanks!

Mark
